src/services/git/__MOCKS__/gitHubClientMockFolder/getContributorsStatsResponse.mock.ts

```diff
@@ -1,3 +1,5 @@
+import { UserItem } from '../../../../../test/helpers/gitHubNock';
```

Comment:
What about using some alias when the path is that long? But it's just nice to have especially in MOCKS. So maybe ignore this comment :D

Reply:
Hm, why not. Do we use aliases elsewhere (for my inspiration)?

Reply:
Hm, why not. @adelkahomolova, do we use aliases elsewhere (for my inspiration)?

Reply:
I don't think so. But I read this article a few days ago. https://dev.to/larswaechter/path-aliases-with-typescript-in-nodejs-4353

Reply:
I have tried that and had some errors. ofc it´d be great

Reply:
I tried my best but still not able to make it work... Not even with the `~` symbol. Can you try that, please? @adelkahomolova, @prokopsimek

Reply:
It's not so important. If it doesn't work, leave it as is.

Reply:
There are more important things I have to do on DX scanner. This is nice to have, I'll have it on my mind and try it if I'll have nothing to do :)